sonarqube-companion-frontend/src/app/config/member/members-settings.component.ts

```diff
@@ -97,6 +107,14 @@ export class MembersSettingsComponent implements OnInit {
       this.loaded = true;
       this.validation.next({valid: false, message: 'Unable to load items'});
     });
+
+    this.loadIntegrationsSummary();
```

Comment:
wow, cóż za łańcuszek. ngoninit (nie rob nic wolaj dalej), load (zrob cos ale i tak wolaj dalej), load integration (cos tam zrob)

a) observable do widoku + async pipe
b) trzymaj poziomy abstrakcji bo mieszasz, wywolania metod logicznych przeplecione z utawianiem flag ladowania

Reply:
Coś tam pozmieniałem. Jak dam observabla z serwisu do widoku z asynciem to connection resety dostanę. Httpclient po wywołaniu robi complete.